vomsimporter.py

```diff
@@ -168,6 +168,10 @@ def has_voms_group(self, voms_group):
 
     def find_group_by_name(self, group_name):
 
+        for name, group in self._iam_groups.items():
```

Comment:
With python dictionary I would not loop over all records but directly ask for item with specific `name`
```python
if name in self._iam_groups:
    return self._iam_groups[name]
```

Reply:
Ok, thanks for the hint.